source/time_tables/passages.cpp

```diff
@@ -176,9 +177,19 @@ void passages(PbCreator& pb_creator,
         auto arrival_date = navitia::to_posix_timestamp(dt_stop_time.first, pb_creator.data);
         passage->mutable_stop_date_time()->set_departure_date_time(departure_date);
         passage->mutable_stop_date_time()->set_arrival_date_time(arrival_date);
+
+        //find base datetime
+        auto base_st = get_base_stop_time(dt_stop_time.second);
+        if (base_st != nullptr) {
+            auto base_ptime = navitia::to_posix_time(dt_stop_time.first, pb_creator.data);
+            auto base_dep_dt = get_base_dt(dt_stop_time.second, base_st, base_ptime, true);
+            passage->mutable_stop_date_time()->set_base_departure_date_time(navitia::to_posix_timestamp(base_dep_dt));
+            passage->mutable_stop_date_time()->set_base_arrival_date_time(navitia::to_posix_timestamp(base_dep_dt));
```

Comment:
minor: It would be nice to give it another name than departure, maybe just `base_dt`
And it's worth doing the same above with `departure_date` and `arrival date`